package.json

```diff
@@ -46,5 +42,8 @@
         "./src/scss"
       ]
     }
+  },
+  "dependencies": {
+    "tslib": "^1.9.0"
```

Comment:
Oh, I didn't realize that `tslib` had to be added as a dependency. Somehow I missed this looking over your pr the first time.

I'm actually a little unsure about this. This project had no dependencies before this, and that was almost certainly a design choice. (We don't want consumers to have to install any additional bloat to use our project).

That being said, `tslib` really doesn't seem like "bloat". It doesn't have any dependencies itself. It's purely a library that reduces our build size by implementing a few functions that typescript would have to define anyways.

That, and it seems every `@angular` package & `rxjs` include the project as a dependency. Referencing this library as a dependency would likely slightly reduce the total build size of any consumers as well.

I think I stand more towards adding this dependency, I just wanted to note my thoughts here for posterity.